docs/cluster-setup-guide/deploy-cluster/sysadmin-deploy-on-prem/linux-packages.rst

```diff
@@ -77,18 +77,30 @@ Install PostgreSQL using ``apt`` or ``yum``
       sudo systemctl enable postgresql.service
 
 #. Configure a system account that Determined will use to connect to PostgreSQL. For example, to use
-   the default ``postgres`` user but update its password:
+   a user named ``determined`` with the password ``determined-password``:
 
    .. code::
 
       sudo -u postgres psql postgres
-      postgres=# \password postgres
+      postgres=# create user determined with encrypted password 'determined-password';
 
-#. Finally, create a database for Determined's use.
+#. The authentication methods enabled vary depending upon the provider of your PostgreSQL
```

Comment:
To me, "The authentication methods enabled" implies that something automatically enables them.  I also think that "upon" should be changed to "on".  If it is the user's responsibility to enable them, perhaps something like this may be clearer:

```
The authentication methods that you specify vary depending on the provider of your PostgreSQL
   distribution.
```

Reply:
> To me, "The authentication methods enabled" implies that something automatically enables them. I

Yes, that is what I mean.    We have observed some installations to only enable `ident` which means that any password authentication doesn't work when Determined connects via localhost.

> I also think that "upon" should be changed to "on". 
ok

Reply:
Something does automatically enable them; the distro packagers choose how to configure hba.conf. :)

The authentication methods configured by default vary....

Reply:
I'd argue that upon is the more formal, and should probably be the one used in docs, but for consistency, it does look like 'on' is the clear winner:
```
sauer@host:~/determined/determined$ grep -E 'depend(ing|s)? +(on|upon)' -oR docs | grep -Eo '(upon|on)' | sort | uniq -c
     88 on
      3 upon
```